mods/ts/rules/shared-vehicles.yaml

```diff
@@ -130,6 +130,59 @@ LPST:
 			gdi: lpst.gdi
 			nod: lpst.nod
 
+LPST2:
+	Inherits: ^Vehicle
+	Valued:
+		Cost: 950
+	Tooltip:
+		Name: Awesomer Mobile Sensor Array
+	Buildable:
+		Queue: Vehicle
+		BuildPaletteOrder: 100
+		Prerequisites: ~factory, radar
+	Health:
+		HP: 600
+	Armor:
+		Type: Light
+	Mobile:
+		Speed: 85
+		ROT: 5
+	RevealsShroud:
+		Range: 10c0
+	-AppearsOnRadar:	# TODO: Discuss whether we want to remove the trait. The original had RadarInvisible=yes
+	DeployToUpgrade:
+		Upgrades: deployed
+		DeployAnimation: make
+		Facing: 160
+		AllowedTerrainTypes: Clear, Road, DirtRoad, Rough
+		DeploySound: place2.aud
+		UnDeploySound: clicky1.aud
+	RenderVoxels:		# Remove once this replaces LPST
+		Image: lpst
+	WithVoxelBody:
+		Image: lpst
+		UpgradeTypes: deployed
+		UpgradeMaxEnabledLevel: 0
+	RenderSprites:		# Remove once this replaces LPST
+		Image: lpst2.gdi
+		FactionImages:
+			gdi: lpst2.gdi
+			nod: lpst2.nod
+	WithSpriteBody@deployed:
+		StartSequence: make
+		UpgradeTypes: deployed
+		UpgradeMinEnabledLevel: 1
+	DisableUpgrade:
+		UpgradeTypes: deployed
+		UpgradeMinEnabledLevel: 1
+		UpgradeMaxAcceptedLevel: 2
```

Comment:
You shouldn't need these MaxAccepted: 2 statements - you're not even checking for them at all during revoking anyway.